Sources/Segment/Utilities/Storage.swift

```diff
@@ -241,23 +243,20 @@ extension Storage {
         }
         
         syncQueue.sync {
-            do {
-                let jsonString = event.toString()
-                if let jsonData = jsonString.data(using: .utf8) {
-                    let handle = try FileHandle(forWritingTo: storeFile)
-                    handle.seekToEndOfFile()
-                    // prepare for the next entry
-                    if newFile == false {
-                        handle.write(",".data(using: .utf8)!)
-                    }
-                    // write the data
-                    handle.write(jsonData)
-                    handle.closeFile()
-                } else {
-                    assert(false, "Storage: Unable to convert event to json!")
+            let jsonString = event.toString()
+            if let jsonData = jsonString.data(using: .utf8) {
+                fileHandle?.seekToEndOfFile()
+                // prepare for the next entry
+                if newFile == false {
+                    fileHandle?.write(",".data(using: .utf8)!)
```

Comment:
Do we absolutely know 100% of the time this explicit will work on all platforms? Makes me uneasy.

Reply:
Why would it not work on all platforms??  It's almost identical to what it was before and tests are passing hunky dory.

Reply:
I guess it just being a comma should be okay.